src/main/java/codesquad/springcafe/controller/LoginController.java

```diff
@@ -0,0 +1,50 @@
+package codesquad.springcafe.controller;
+
+import codesquad.springcafe.model.User;
+import codesquad.springcafe.service.LoginService;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class LoginController {
+
+    private final LoginService loginService;
+    private static final Logger log = LoggerFactory.getLogger(LoginController.class);
+
+    public LoginController(LoginService loginService) {
+        this.loginService = loginService;
+    }
+
+    @GetMapping("/login")
+    public String loginForm() {
+        return "users/login";
+    }
+
+    @PostMapping("/login")
+    public String login(@ModelAttribute User user, HttpServletRequest request) {
```

Comment:
```suggestion
    public String login(@ModelAttribute User user, HttpSession session) {
```

request보다 직접 세션을 가져오는 게 더 편합니다.